tests/accumulator/tests.c

```diff
@@ -0,0 +1,210 @@
+/* Copyright 2019 Outscale SAS
+ *
+ * This file is part of Packetgraph.
+ *
+ * Packetgraph is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published
+ * by the Free Software Foundation.
+ *
+ * Packetgraph is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Packetgraph.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <packetgraph/packetgraph.h>
+#include "utils/tests.h"
+#include <packetgraph/accumulator.h>
+#include "brick-int.h"
+#include "packets.h"
+#include "utils/mempool.h"
+#include "utils/bitmask.h"
+#include "collect.h"
+#include "packetsgen.h"
+
+#define NB_PKTS 60
+
+static void realloc_pkts(struct rte_mbuf **packets)
+{
+	for (int i = 0; i < NB_PKTS; i++) {
+		packets[i] = rte_pktmbuf_alloc(mp);
+		g_assert(packets[i]);
+		packets[i]->udata64 = i;
+	}
+}
+
+static void accumulator_packets_poll(enum pg_side dir, uint64_t loop)
+{
+	struct pg_brick *acc, *col1, *col2;
+	struct pg_error *error = NULL;
+	struct rte_mbuf *packets[NB_PKTS];
+	struct rte_mbuf **res_pkts[PG_MAX_SIDE];
+	uint16_t packet_count;
+	uint64_t filtered_mask;
+
+	memset(packets, 0, NB_PKTS * sizeof(struct rte_mbuf *));
+	realloc_pkts(packets);
+	g_assert(!error);
```

Comment:
realloc_pkts doesn't seems to able  to set error, so it's not useful to check it here